appveyor.yml

```diff
@@ -0,0 +1,26 @@
+version: 1.0.{build}
+environment:
+  matrix:
+  - APPVEYOR_BUILD_WORKER_IMAGE: ubuntu
+    Purpose: preview
+  - APPVEYOR_BUILD_WORKER_IMAGE: ubuntu
+    Purpose: stable
+
+install:
+#  - ps: Install-module pester -Scope CurrentUser -Force -SkipPublisherCheck
```

Comment:
nit: any reason for this commented out code?

Reply:
removed